### PR TITLE
UK Diet Scores Playback

### PR DESCRIPTION
**Pull request by selvach**
- Switch "Experiments" to use the redux user state that @petermason created for the drawer menu.
- Merge's contents of 2 DietScores screens to one. (Contents of DietStudyPlaybackGlobal now appears on on DietStudyPlaybackIntro)
- Switches Avatars
   - US has Christopher + Sarah. 
   - UK has Sarah only
- Switches weblink based on country
- A/B Test the Callout Card to start DietScore play back. 50% on the UK Dashboard. 50% on UK ThankYou Page.

### CODE REVIEW

File: src/features/diet-study-playback/DietStudyPlaybackCoordinator.ts [hunk 1]
```diff
@@ -56,6 +53,12 @@ class DietStudyPlaybackCoordinator extends Coordinator {
   signUpToNewsletter(signup: boolean) {
     return this.contentService.signUpForDietNewsletter(signup);
   }
+
+  getDietStudyInfoUrl = () => {
+    return isUSCountry()
+      ? 'https://covid.joinzoe.com/us-post/covid-diet-feedback'
+      : 'https://covid.joinzoe.com/post/covid-lockdown-diet';
```
Comment (bionara) [inline src/features/diet-study-playback/DietStudyPlaybackCoordinator.ts:60]: heads up - this link is a 404 (the uk one, https://covid.joinzoe.com/post/covid-lockdown-diet )

File: src/features/diet-study-playback/DietStudyPlaybackIntroScreen.tsx [hunk 2]
```diff
@@ -5,56 +5,153 @@ import { Image, SafeAreaView, ScrollView, StyleSheet, View } from 'react-native'
...
+          {IntroCard}
 
-          <BasicCard>
+          <BasicCard style={{ marginBottom: 24 }}>
             <View style={styles.quoteIcon}>
               <QuoteMarks />
             </View>
-            <Header3Text style={{ marginBottom: 20 }}>
+            <Header3Text style={{ marginBottom: 24 }}>
               We are excited to share our insights on how diets have changed during the pandemic.
```
Comment (bionara) [inline src/features/diet-study-playback/DietStudyPlaybackIntroScreen.tsx:95]: Are these fine as text and not translations?

File: src/features/diet-study-playback/DietStudyPlaybackIntroScreen.tsx [hunk 3]
```diff
@@ -5,56 +5,153 @@ import { Image, SafeAreaView, ScrollView, StyleSheet, View } from 'react-native'
...
+            <HeaderText style={styles.titleText}>Global diet changes during the pandemic</HeaderText>
+            <SecondaryText style={styles.subtext}>
+              From 1.5 million app users surveyed across the US and the UK, we saw that the pandemic has affected our
+              diets very differently.
+            </SecondaryText>
+            <Header3Text style={styles.subtitle}>
+              4 in 10 people changed their diet quality during the pandemic.
+            </Header3Text>
+            <Image style={styles.infoCards} source={dietStudyPlaybackGlobal1} />
+            <Header3Text style={styles.subtitle}>Some people increased their alcohol consumption.</Header3Text>
```
Comment (bionara) [inline src/features/diet-study-playback/DietStudyPlaybackIntroScreen.tsx:112]: 🥇

**Review by bionara — DISMISSED** [review]
Code looks good! pulled locally and reviewing.

File: src/features/diet-study-playback/DietStudyPlaybackCoordinator.ts (hunk 1 above)
Comment (selvach) [reply src/features/diet-study-playback/DietStudyPlaybackCoordinator.ts:60]: Thanks! I need to remind Chris to publish the blog!

**Review by bionara — APPROVED** [review]

**Pull request merged**